packages/@lightningjs/ui-components/src/components/ListItem/ListItemSlider.test.js

```diff
@@ -92,7 +92,7 @@ describe('ListItemSlider', () => {
   it('should not exist Value if value not exist', () => {
     listItemSlider.value = null;
     testRenderer.forceAllUpdates();
-    expect(listItemSlider._Value).toBeUndefined();
+    expect(listItemSlider._Value).toBeDefined();
```

Comment:
I think we'll want this to still be undefined, since that is what the test originally was checking for. We can get this to pass if we wrap `this._updateValue();` in `_update()` in a `setTimeout`. So it would be:

```js
_update() {
  super._update();
  this._updateSliderPosition();
  setTimeout(() => {
    this._updateValue();
  }, 0);
}
```